sys/include/libufs.h

```diff
@@ -10,11 +10,22 @@
 
 
 struct Chan;
-struct mount;
 struct vnode;
+struct thread;
 
+/* Wrapper for a UFS mount.  Should support reading from both kernel and user
+ * space (eventually)
+ */
+struct mount {
+	struct ufsmount*	mnt_data;
```

Comment:
we need to start making this more like Plan 9 code.
typedef Struct ufsmount *Ufsmount
then declare it as Ufsmount *mnt_data;